source/removals.rst

```diff
@@ -0,0 +1,90 @@
+.. _removals:
+
+===========================
+ Other miscellaneous ideas
+===========================
+
+Dropping Python 2 support
+=========================
+
+With Python 2.7 reaching its supported end-of-life in 2020, like some other
+Python projects (e.g. IPython / Jupyter) we should seriously contemplate making
+pandas 2.0 only support Python 3.5 and higher. In addition to lowering the
```

Comment:
given that I'd like to see pandas 2.0 revamp its internal string handling, doing this work for both Python 2 and 3 seems like not a good use of time. would be great to remove all the compatibility code, too (and potentially start making use of some python 3 only features, like asyncio)